src/filter/distort0r/distort0r.c

```diff
@@ -66,6 +69,16 @@ void f0r_get_param_info(f0r_param_info_t* info, int param_index)
       info->type = F0R_PARAM_DOUBLE;
       info->explanation = "The frequency of the plasma signal";
       break;
+    case 2:
+      info->name = "Mode";
+      info->type = F0R_PARAM_BOOL;
+      info->explanation = "\'Time Based\' or \'Adjustable Velocity\'";
```

Comment:
Unfortunately, Mode with this explanation does not make it clear which value determines which mode. The quickest fix is to improve the explanation. Alternatively, Mode could be a string param that takes "time" or "velocity" as values, or it could be renamed to "Use Velocity."